scripts/composer/ScriptHandler.php

```diff
@@ -40,7 +41,7 @@ public static function createRequiredFiles(Event $event) {
       $fs->copy($drupalRoot . '/sites/default/default.settings.php', $drupalRoot . '/sites/default/settings.php');
       require_once $drupalRoot . '/core/includes/bootstrap.inc';
       require_once $drupalRoot . '/core/includes/install.inc';
-      $settings['config_directories'] = [
+      $settings['settings'] = [
```

Comment:
@orakili just checking what we want here (the version we're changing from had the wrong path so I'm not even sure if this line is necessary) - on IASC the change is https://github.com/UN-OCHA/iasc8/pull/504/commits/069b1839273af82d47be6204dd5973db87d52abb
Can you confirm that this is the right change?

Reply:
@lazysoundsystem that's correct for Drupal 9. See the example on https://api.drupal.org/api/drupal/core%21includes%21install.inc/function/drupal_rewrite_settings/9.0.x and the change from Drupal 8.8: https://www.drupal.org/docs/configuration-management/changing-the-storage-location-of-the-sync-directory#s-syntax-changes-in-drupal-880

**But** it doesn't matter because we override the settings.php file in https://github.com/UN-OCHA/slt8-site/blob/2c6197d0816caa5bb5e85d619b39c9b692eff6ad/docker/Dockerfile#L29.

Reply:
Thanks @orakili, I think that means we can remove the line completely, right?

We're going to face this change on all the sites so it's good to get it right!

Reply:
@lazysoundsystem I think all the settings.php block can be removed:  https://github.com/UN-OCHA/slt8-site/blob/ca14bae81f431c62e7138e1cbe0ffada807f89df/scripts/composer/ScriptHandler.php#L39-L53